### PR TITLE
add Sitemap Images

### PR DESCRIPTION
**Pull request by SunwelLight**
Added a new class for the generation of site map images.

### CODE REVIEW

**Comment by samdark** [comment]
Overall method overrides are copying too much while changing not so much. It makes sense to extract these into protected methods in the Sitemap class.

**Comment by agoestri-tkpd** [comment]
Please merge, I need this feature

**Comment by SunwelLight** [comment]
The duplicating method is removed

File: Sitemap.php [hunk 1]
```diff
@@ -164,6 +170,9 @@ private function createNewFile()
         if ($this->useXhtml) {
```
Comment (samdark) [inline Sitemap.php]: It's better to extract writing whole XML header into separate protected method and override it in ImageSitemap. Sitemap base class should not know about images at all.

File: SitemapImages.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
+<?php
+namespace samdark\sitemap;
+
+/**
+ * A class for generating image Sitemaps (http://www.sitemaps.org/)
+ *
+ * @author SunwelLight <sunwellight@gmail.com>
+ */
+class SitemapImages extends Sitemap
```
Comment (samdark) [inline SitemapImages.php]: I'd rename to ImageSitemap

**Comment by samdark** [comment]
@SunwelLight overall it looks good but last two little changes are needed. Also need changes to README.md explaining image sitemap...

**Comment by samdark** [comment]
Going to push a new version that implements it differently.

**Pull request closed without merging**